README.rst

```diff
@@ -120,16 +120,16 @@ The plugin itself holds a default config that can be used for any Coverity proje
         'stream': 'some_coverty_stream',
     }
 
-This default configuration build into the plugin, can be overriden through the conf.py of your project.
+This default configuration, built into the plugin, can be overridden through the *conf.py* of your project.
```

Comment:
No comma here after configuration - its the same sentence.

Reply:
I made it a relative clause for more clarity. 24ff2f805b9c3789c10252c0f528d871ab0face2